package.json

```diff
@@ -1,35 +1,47 @@
 {
-  "name": "ninja-util",
-  "version": "1.2.3",
-  "description": "Helper library to develop Node.JS applications",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/ibm-hyperknowledge/ninja-util"
-  },
-  "author": "IBM Research",
-  "license": "MIT",
-  "main": "index.js",
-  "contributors": [
-    {
-      "name": "Polyana Costa",
-      "email": "polyana.bezerra@ibm.com"
+    "name": "ninja-util",
+    "version": "1.2.5",
```

Comment:
@srfiorini the last version was 1.2.3. The changes that I've made should update the version to 1.2.4.

However, I believe that your changes should change the MINOR level, does it make sense to move to 1.3.0 ?